documentation/intro-dylan/source/modules-libraries.rst

```diff
@@ -153,7 +154,7 @@ use a similar syntax:
 
     define open generic sample-function (o :: <object>) => ();
 
-A third form, ``define sealed domain``, partially
+A third form, :drm:`define sealed domain <define_sealed_domain>`, partially
```

Comment:
I wonder if it would be too magic to change https://github.com/dylan-lang/sphinx-extensions/blob/master/sphinxcontrib/dylan/domain/drmindex.py#L274 to replaces spaces with underscores so we don't have to repeat like this.  Just an idea.

Reply:
I agree here too,

Reply:
Ok, I take note.

Reply:
This change will affect [drm_link](https://github.com/dylan-lang/sphinx-extensions/blob/321d4e77302b1fff1f25ea8fe32272dd60e628f6/sphinxcontrib/dylan/domain/dylandomain.py#L37) and the validation of invalid syntax in `:dylan:drm` roles. Is that ok?